### ISSUE TITLE
Unexpected image name from download image

### ISSUE THREAD

**Issue by conformist-mw** [lk-geimfari/mimesis#490]
Hi there! 
If download images with `download_image` and `internet.stock_image` files saves with name of get param:
`https://source.unsplash.com/300x300/?people` split by last slash → filename = `?people`.

**Comment by lk-geimfari** [comment]
Jurek Dubovik expressed a desire to fix this issue. So, Jurek if you've read this comment and you've saved your interest to solve this you can start right now.

Just open terminal and do this:
```zsh
# Fork this repository and clone it, like that:
git clone git@github.com:your-github-username/mimesis.git
# Go to the project folder
cd mimesis/
# Create new branch
git checkout -b fixed_image_downloader_issue
# Add feature
# ...
git commit -m "Fixed issue with names on downloading images"
git push your_branch 
```

After that: https://help.github.com/articles/creating-a-pull-request/

**Comment by conformist-mw** [comment]
@lk-geimfari I've already fixed this function in my code so I create a [pull request](https://github.com/lk-geimfari/mimesis/pull/499) if you don't mind.

**Comment by lk-geimfari** [comment]
@conformist-mw Sure!

**Comment by lk-geimfari** [comment]
Fixed in #499
